change/office-ui-fabric-react-2020-02-24-17-30-43-master.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "Add polling option for the Keytip Manager",
```

Comment:
The comment here shows up in our release notes. Can you clean this up to be readable to a consumer?

KeytipManager: Adjusting how keytip data is sent to the manager to optimize rendering times for components which use keytips.

Reply:
Will do!